src/app/calculator/motors/full-load-amps/full-load-amps-form/full-load-amps-form.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { Settings } from '../../../../shared/models/settings';
+import { FLAMotorWarnings, FullLoadAmpsService } from '../full-load-amps.service';
+import { SettingsDbService } from '../../../../indexedDb/settings-db.service';
+import { Subscription } from 'rxjs';
+import { FanMotor } from '../../../../shared/models/fans';
+import { motorEfficiencyConstants } from '../../../../psat/psatConstants';
+
+@Component({
+  selector: 'app-full-load-amps-form',
+  templateUrl: './full-load-amps-form.component.html',
+  styleUrls: ['./full-load-amps-form.component.css']
+})
+export class FullLoadAmpsFormComponent implements OnInit {
+  @Input()
+  settings: Settings;
+  form: FormGroup;
+  formWidth: number;
+  idString: string;
+  fullLoadAmps: number;
+
+  formValid: boolean;
+  warnings: FLAMotorWarnings;
+  motor: FanMotor;
+  resetDataSub: Subscription;
+  generateExampleSub: Subscription;
+  fullLoadAmpsResultSub: Subscription;
+
+  frequencies: Array<number> = [
+    50,
+    60
+  ];
+  efficiencyClasses: Array<{ display: string, value: number }>;
+  
+  constructor(private settingsDbService: SettingsDbService, private fullLoadAmpsService: FullLoadAmpsService) { }
+
+  ngOnInit(): void {
+    this.settings = this.settingsDbService.globalSettings;
```

Comment:
Can remove settings assignment and the injected service. It's being set in the parent full-load-amps.ts just needs to be passed down to this component